amica/scripts/screenshot.ts

```
@@ -0,0 +1,370 @@
+
+    if (value === undefined && key !== 'help') {
+      console.error(`Error: Missing value for argument --${key}`);
+      process.exit(1);
+    }
+
+    switch (key) {
+      case 'output':
+        options.output = value;
+        break;
+      case 'width':
+        const width = parseInt(value, 10);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)